### DISCUSSION TITLE
Please pic a name for the manager/installer

### DISCUSSION THREAD

**Discussion by smf-gitwit** [microsoft/winget-cli#2167] [Ideas] [answered]
Microsoft.DesktopAppInstaller_8wekyb3d8bbwe? Does it only install Desktop apps? No.

Windows Package Manager?

WinGet?

WinGet is a CLI app why have Winget-CLI in the name? Chi is tea. Scampi is shrimp. Is there a Winget-GUI no so why the name?

"8wekyb3d8bbwe" this is what exactly? Alphanumeric salad.

Pick a pronoun and stick with it.

**Comment by denelon** [comment, marked as answer]
The Windows Package Manager is distributed via the Microsoft Store inside the "[App Installer](https://apps.microsoft.com/store/detail/app-installer/9NBLGGH4NNS1)".
Microsoft.DesktopAppInstaller_8wekyb3d8bbwe is the name of the "Desktop App Installer" (an original internal name) MSIX bundle generated by the build pipeline.
The "App Installer" is more than just the Windows Package Manager.

The "alphanumeric salad" at the end is generated by the Microsoft Store publishing process and is generally not directly exposed to consumer end users.

The way invoking "winget" is mapped to the actual executable is handled via an App Execution Alias. Originally, the cli was named appinstallercli.exe which aligned to the original package name, but in one of our more recent releases, the name was changed to winget.exe.

Renaming the packaged asset for GitHub releases could cause even more confusion for those who are aware of the underlying technical details. Unfortunately, there is rarely "one right answer". In most cases there are "many correct answers".

>[There are only two hard things in Computer Science: cache invalidation and naming things.
-- Phil Karlton](https://martinfowler.com/bliki/TwoHardThings.html)
